build-tools/automation/azure-pipelines-oss.yaml

```diff
@@ -2,66 +2,186 @@
+      displayName: make jenkins
+
+    - script: |
+        echo "make create-installers V=1 CONFIGURATION=$(XA.Build.Configuration)"
+        make create-installers V=1 CONFIGURATION=$(XA.Build.Configuration)
+      workingDirectory: $(Build.SourcesDirectory)
+      displayName: create installers
+
+    - script: |
+        echo "make package-oss CONFIGURATION=$(XA.Build.Configuration) V=1"
+        make package-oss CONFIGURATION=$(XA.Build.Configuration) V=1
+      workingDirectory: $(Build.SourcesDirectory)
+      displayName: package oss
+
+    - script: |
+        echo "all-tests CONFIGURATION=$(XA.Build.Configuration) V=1"
+        make all-tests CONFIGURATION=$(XA.Build.Configuration) V=1
+      workingDirectory: $(Build.SourcesDirectory)
+      displayName: make all-tests
+      condition: and(succeeded(), eq(variables['BUILD_TESTS'], 'true'))
+
+    - script: |
+        echo "make package-build-status CONFIGURATION=$(XA.Build.Configuration) V=1"
+        make package-build-status CONFIGURATION=$(XA.Build.Configuration) V=1
+      workingDirectory: $(Build.SourcesDirectory)
+      displayName: package build status
+
+    - task: MSBuild@1
+      displayName: pack all nupkgs
+      inputs:
+        solution: $(Build.SourcesDirectory)/build-tools/create-packs/Microsoft.Android.Sdk.proj
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /t:CreateAllPacks /restore /bl:$(Build.SourcesDirectory)/bin/Build$(XA.Build.Configuration)/create-all-packs.binlog
+
+    - task: PublishPipelineArtifact@1
+      displayName: upload nupkgs
+      inputs:
+        artifactName: $(NuGetArtifactName)
+        targetPath: $(Build.SourcesDirectory)/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)
+
+    - script: |
+        exitCode=0
+        make run-performance-tests CONFIGURATION=$(XA.Build.Configuration) V=1 || exitCode=$?
+        if [[ $exitCode -ne 0 ]]; then
+          echo "ERROR: run-performance-tests FAILED, status: ${exitCode}"
+          exit $exitCode
+        fi
+      workingDirectory: $(Build.SourcesDirectory)
+      displayName: run performance tests
+      condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
+
+    - template: yaml-templates/upload-results.yaml
+      parameters:
+        artifactName: Build Results - macOS
+
+- stage: linux_stage
+  displayName: Linux
+  condition: eq(variables['EnableLinuxStage'], 'true')    # The variable is defined on the pipeline definition
+  dependsOn: []                 # Run stage in parallel
+  jobs:
+  - job: linux_build_package
+    displayName: Linux Build
+    pool: $(XA.Build.LinuxOSSPool)                        # The variable is defined on the pipeline definition
+    timeoutInMinutes: 180
+    cancelTimeoutInMinutes: 5
+    workspace:
+      clean: all
+    steps:
+    - checkout: self
+      submodules: recursive
+
+    - script: |
+        # Report used vs. available disk space on each volume
+        df -H
+      displayName: report disk info
```

Comment:
I think we can remove this script, I'm working to address this issue in https://github.com/xamarin/xamarin-android/pull/4674

Reply:
Removed. Commit: e6d8876378c36b290aadbbcfcd7ad9058307b0ef